### PR TITLE
[WIP] run-graphical-haskell.hs: support language pragmas

### PR DESCRIPTION
**Pull request by vaibhavsagar**

### CODE REVIEW

File: run-graphical-haskell.hs [hunk 1]
```diff
@@ -22,22 +23,25 @@ main =
...
     withSystemTempFile "declarations-.hs" $ \declsPath declsHandle ->
     withSystemTempFile "graphical-haskell-.hs" $ \hsPath hsHandle ->
       (
         IO.hClose hsHandle >>
-        traverse_ (writeCode importsHandle declsHandle) (nodeCodeBlocks svgRoot) >>
+        traverse_ (writeCode pragmasHandle importsHandle declsHandle) (nodeCodeBlocks svgRoot) >>
+        IO.hClose pragmasHandle >>
         IO.hClose importsHandle >>
         IO.hClose declsHandle >>
         Proc.callCommand (unwords ["cat", importsPath, declsPath, ">", hsPath]) >>
```
Comment (chris-martin) [inline run-graphical-haskell.hs]: got to cat the pragmas file in here too

Comment (vaibhavsagar) [reply run-graphical-haskell.hs]: Whoops! Will update.

**Comment by chris-martin** [comment]
Cool! Merged and cleaned up a few things.

The svg text nodes got scaled out of proportion so they were stretched vertically in your commit. Did you edit with Inkscape?

**Comment by vaibhavsagar** [comment]
I did! I must have done something wrong.

**Pull request merged**